lnrpc/stateservice.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package lnrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc";
+
+/*
+ * Comments in this file will be directly parsed into the API
+ * Documentation as descriptions of the associated method, message, or field.
+ * These descriptions should go right above the definition of the object, and
+ * can be in either block or // comment format.
+ *
+ * An RPC method can be matched to an lncli command by placing a line in the
+ * beginning of the description in exactly the following format:
+ * lncli: `methodname`
+ *
+ * Failure to specify the exact name of the command will cause documentation
+ * generation to fail.
+ *
+ * More information on how exactly the gRPC documentation is generated from
+ * this proto file can be found here:
+ * https://github.com/lightninglabs/lightning-api
+ */
+
+// State service is a always running service that exposes the current state of
+// the wallet and RPC server.
+service State {
```

Comment:
Open for a better name for the service than just `State`, if any suggesitons 😅

Reply:
I don't have any objection to a generic name. As long as we can add more "sub" states to the `SubscribeStateResponse` it doesn't make any sense to choose a more specific name.
For example, the leader election state implemented in https://github.com/lightningnetwork/lnd/pull/4998 could be added here as well which is nice!

Reply:
Indeed this is a nice candidate to add leader status!